ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -165,7 +169,7 @@ tests:
       - env: HYPERSHIFT_RELEASE_LATEST
         name: hypershift-operator
       - env: OCP_IMAGE_LATEST
-        name: release:latest
+        name: release:previous
```

Comment:
This means that the PR isn't tested anymore, previous doesn't `nclude_built_images: true`

Reply:
Thanks, fixed.
The goal is to have the infra cluster in openshift-ci deployed with ocp 4.11.
/cc @davidvossel